examples/src/components/MultipleDomainsExample.jsx

```diff
@@ -0,0 +1,91 @@
+// (C) 2007-2018 GoodData Corporation
+import React, { Component } from 'react';
+import { keyBy, mapValues } from 'lodash';
+import { Kpi } from '@gooddata/react-components';
+import { factory } from 'gooddata';
+import '@gooddata/react-components/styles/css/main.css';
+
+import { totalSalesIdentifier, projectId } from '../utils/fixtures';
+
+export class MultipleDomainsExample extends Component {
+    constructor() {
+        super();
+        this.state = {
+            sdk2Config: { projectId: '', measureId: '' },
+            sdk2LoginError: '',
+            sdk2: false
+        };
+        this.setSdk2 = this.setSdk2.bind(this);
+    }
+
+    setSdk2(e) {
+        const formRef = e.target;
+        const inputs = formRef.querySelectorAll('[name]');
+        const inputsByName = keyBy(inputs, 'name');
+        const formValues = mapValues(inputsByName, 'value');
+
+        const sdk2 = factory({ domain: formValues.domain });
+        sdk2.user.login(formValues.sdk2Username, formValues.sdk2Password)
+            .then(() => {
+                this.setState({ sdk2LoginError: '' });
+                this.setState({ sdk2, sdk2Config: formValues });
+            }).catch((error) => {
+                this.setState({ sdk2LoginError: error.toString() });
+                console.log(error); // eslint-disable-line no-console
+            });
+
+        e.preventDefault();
+        return false;
+    }
+
+    render() {
+        return (
+            <div>
+                <h2>Default SDK:</h2>
+                <Kpi
+                    projectId={projectId}
+                    measure={totalSalesIdentifier}
+                />
+
+                <hr className="separator" />
+                <h2>Second SDK:</h2>
+                <Kpi
+                    projectId={this.state.sdk2Config.projectId}
+                    measure={this.state.sdk2Config.measureId}
+                    sdk={this.state.sdk2}
+                />
+
+                <div className="alert">{this.state.sdk2LoginError}</div>
+
+                <form action="?" method="get" onSubmit={this.setSdk2}>
+                    <p><strong>Login to second SDK first:</strong></p>
+                    Careful - cookies ignore port, you have to use different hostname to log in.<br />
+
+                    <table>
+                        <tbody>
+                            <tr><td>Domain: </td><td><input type="text" name="domain" /></td></tr>
```

Comment:
even with all the comments I am still not clear what to put in here.
Do I have to make another server on localhost and expose it on different IP?
Do giraffes accept CORS requests?